x/concentrated-liquidity/incentives_test.go

```diff
@@ -194,20 +194,15 @@ func expectedIncentivesFromRate(denom string, rate sdk.Dec, timeElapsed time.Dur
 // Returns value as truncated sdk.Coins as the primary use of this helper is testing higher level incentives functions such as claiming.
 func expectedIncentivesFromUptimeGrowth(uptimeGrowths []sdk.DecCoins, positionShares sdk.Dec, timeInPool time.Duration, multiplier sdk.Int) sdk.Coins {
 	// Sum up rewards from all inputs
-	totalRewards := sdk.DecCoins(nil)
+	totalRewards := sdk.Coins(nil)
 	for uptimeIndex, uptimeGrowth := range uptimeGrowths {
 		if timeInPool >= types.SupportedUptimes[uptimeIndex] {
-			totalRewards = totalRewards.Add(uptimeGrowth...)
+			curRewards := uptimeGrowth.MulDecTruncate(positionShares).MulDecTruncate(multiplier.ToDec())
+			totalRewards = totalRewards.Add(sdk.NormalizeCoins(curRewards)...)
```

Comment:
What is the reason we truncate during every loop iteration instead of truncating at the end?

Reply:
Each loop corresponds to claiming tokens from a single accumulator, which truncates `DecCoins` to `Coins`. The previous version of this test helper did not match this behavior, which broke rounding-related edge case tests I added in this PR, so this change just aligns its logic with what we expect to see in core logic